### PR TITLE
Added main page for export templates

### PR DESCRIPTION
**Pull request by MatanHeledPort**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

File: docs/exporters/k8s-exporter/export-templates/export-templates.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+For example, to figure out how to map your ArgoCD resources, you can run:
+
+```bash showLineNumbers
+$ kubectl api-resources | grep -i argo
+applications                      app,apps           argoproj.io/v1alpha1                   true         Application
+applicationsets                   appset,appsets     argoproj.io/v1alpha1                   true         ApplicationSet
+appprojects                       appproj,appprojs   argoproj.io/v1alpha1                   true         AppProject
+```
+
+Then, in order to map `applications`, your `config.yaml` will contain:
```
Comment (MPTG94) [inline docs/exporters/k8s-exporter/export-templates/export-templates.md:23]:
```suggestion
Then, in order to map ArgoCD `applications`, add the matching resource kind to your `config.yaml`:
```

File: docs/exporters/k8s-exporter/export-templates/export-templates.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+```yaml
+- kind: argoproj.io/v1alpha1/applications
+  identifier: .metadata.name
+  blueprint: '"argoApp"'
+  properties:
+  ...
+```
+
+We created some pre-built Port K8s exporter templates covering different applications.
```
Comment (MPTG94) [inline docs/exporters/k8s-exporter/export-templates/export-templates.md:33]:
```suggestion
We created several pre-built Port K8s exporter templates covering different applications.
```

File: docs/exporters/k8s-exporter/export-templates/export-templates.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Each template will include:
```
Comment (MPTG94) [inline docs/exporters/k8s-exporter/export-templates/export-templates.md:34]: Right now this line and the previous one appear in the same line, if you want to separate them, you will need to add another blank link between therm

File: docs/exporters/k8s-exporter/export-templates/argocd-template.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# ArgoCD Template
+
+An ArgoCD Application is an Argo CRD which is responsible for managing different k8s objects, while a ArgoCD project clusters Applications.
```
Comment (MPTG94) [inline docs/exporters/k8s-exporter/export-templates/argocd-template.md:3]:
```suggestion
An ArgoCD Application is an Argo CRD which is responsible for managing different k8s objects, while an ArgoCD project is an Argo CRD which clusters Applications.
```

File: docs/exporters/k8s-exporter/export-templates/argocd-template.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+:::info
+The files to this Exporter Template can be found **[here](https://github.com/port-labs/port-k8s-exporter-use-cases/tree/main/argocd)**
+:::
+
+Using this template, you can easily create `argocdApp` and `argocdAppProject` Blueprints, and configure your K8s exporter to query your ArgoCD resources.
+
+:::note
+Blueprints created in this template define Relations to Blueprints which are referenced in the [Full Kubernetes Exporter](../../../complete-use-cases/full-kubernetes-exporter.md) docs.
+
+This template covers ArgoCD Applications with relations to Deployments and Services, like ArgoCD's [Getting Started](https://argo-cd.readthedocs.io/en/stable/getting_started/) application `guestbook-ui`.
```
Comment (MPTG94) [inline docs/exporters/k8s-exporter/export-templates/argocd-template.md:14]:
```suggestion
This template covers ArgoCD Applications with Relations to Deployments and Services, like ArgoCD's [Getting Started](https://argo-cd.readthedocs.io/en/stable/getting_started/) application `guestbook-ui`.
```

File: docs/exporters/k8s-exporter/export-templates/argocd-template.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+An ArgoCD Application is an Argo CRD which is responsible for managing different k8s objects, while a ArgoCD project clusters Applications.
+
+:::info
+The files to this Exporter Template can be found **[here](https://github.com/port-labs/port-k8s-exporter-use-cases/tree/main/argocd)**
+:::
+
+Using this template, you can easily create `argocdApp` and `argocdAppProject` Blueprints, and configure your K8s exporter to query your ArgoCD resources.
+
+:::note
+Blueprints created in this template define Relations to Blueprints which are referenced in the [Full Kubernetes Exporter](../../../complete-use-cases/full-kubernetes-exporter.md) docs.
```
Comment (MPTG94) [inline docs/exporters/k8s-exporter/export-templates/argocd-template.md:12]:
```suggestion
Blueprints created in this template define Relations to Blueprints which are referenced in the [Full Kubernetes Exporter](../../../complete-use-cases/full-kubernetes-exporter.md) docs.

Specifically the Blueprints in this example are related to the [service](https://github.com/port-labs/k8s-exporter-example/blob/main/terraform/Service.tf) and [deployment](https://github.com/port-labs/k8s-exporter-example/blob/main/terraform/Deployment.tf) Blueprints.
```

File: docs/exporters/k8s-exporter/export-templates/argocd-template.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# ArgoCD Template
+
+An ArgoCD Application is an Argo CRD which is responsible for managing different k8s objects, while a ArgoCD project clusters Applications.
+
+:::info
+The files to this Exporter Template can be found **[here](https://github.com/port-labs/port-k8s-exporter-use-cases/tree/main/argocd)**
```
Comment (MPTG94) [inline docs/exporters/k8s-exporter/export-templates/argocd-template.md:6]: Please make the repo link bold in the K8s exporter complete use case doc

**Review by MPTG94 — APPROVED** [review]
LGTM ask Aviv and Ella for a proof read after applying fixes

File: docs/exporters/k8s-exporter/export-templates/argocd-template.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
+# ArgoCD Template
+
+An ArgoCD Application is an Argo CRD which is responsible for managing different k8s objects, while an ArgoCD project is an Argo CRD which clusters Applications.
```
Comment (EllaFurman) [inline docs/exporters/k8s-exporter/export-templates/argocd-template.md:3]:
```suggestion
An ArgoCD application is an Argo CRD which is responsible for managing different k8s objects, while an ArgoCD project is an Argo CRD which clusters applications.
```

File: docs/exporters/k8s-exporter/export-templates/argocd-template.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+:::info
+The files to this Exporter Template can be found **[here](https://github.com/port-labs/port-k8s-exporter-use-cases/tree/main/argocd)**
```
Comment (EllaFurman) [inline docs/exporters/k8s-exporter/export-templates/argocd-template.md:6]:
```suggestion
You can find the files to this exporter template **[here](https://github.com/port-labs/port-k8s-exporter-use-cases/tree/main/argocd)**
```

File: docs/exporters/k8s-exporter/export-templates/export-templates.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+# Export Templates
+
+As shown in the [Mapping Kubernetes resources](../../../complete-use-cases/full-kubernetes-exporter.md) use case, mapping Kubernetes objects to Port is easy using Port's K8s exporter.
+It is also possible to export and map [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) from your K8s cluster using the exporter. To obtain all of the resource types you can query in your cluster, you can run:
+
+```bash showLineNumbers
+$ kubectl api-resources
+```
+
+For example, to figure out how to map your ArgoCD resources, you can run:
```
Comment (EllaFurman) [inline docs/exporters/k8s-exporter/export-templates/export-templates.md:14]:
```suggestion
For example, in order to figure out how to map your ArgoCD resources, you can run:
```

File: docs/exporters/k8s-exporter/export-templates/argocd-template.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+:::
+
+Using this template, you can easily create `argocdApp` and `argocdAppProject` Blueprints, and configure your K8s exporter to query your ArgoCD resources.
+
+:::note
+Blueprints created in this template define Relations to Blueprints which are referenced in the [Full Kubernetes Exporter](../../../complete-use-cases/full-kubernetes-exporter.md) docs.
+
+Specifically the Blueprints in this example are related to the [service](https://github.com/port-labs/k8s-exporter-example/blob/main/terraform/Service.tf) and [deployment](https://github.com/port-labs/k8s-exporter-example/blob/main/terraform/Deployment.tf) Blueprints.
+
+This template covers ArgoCD Applications with Relations to Deployments and Services, like ArgoCD's [Getting Started](https://argo-cd.readthedocs.io/en/stable/getting_started/) application `guestbook-ui`.
```
Comment (EllaFurman) [inline docs/exporters/k8s-exporter/export-templates/argocd-template.md:16]:
```suggestion
This template covers ArgoCD applications with Relations to deployments and services, like ArgoCD's [Getting Started](https://argo-cd.readthedocs.io/en/stable/getting_started/) application `guestbook-ui`.
```

**Review by EllaFurman — CHANGES_REQUESTED** [review]

**Pull request merged**